packages/react-storage/src/components/StorageBrowser/components/Checkbox.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { CLASS_BASE } from '../views/constants';
+import {
+  ViewElement,
+  LabelElement,
+  InputElement,
+} from '../context/elements/definitions';
+
+const BLOCK_NAME = `${CLASS_BASE}__checkbox`;
+const LABEL_CLASSNAME = `${BLOCK_NAME}-label`;
+
+interface CheckboxControlProps {
+  checked: boolean;
+  disabled?: boolean;
+  id: string;
+  labelHidden?: boolean;
+  labelText: string;
```

Comment:
Component level props should always be optional

Reply:
But then the label will have blank text, which would hurt accessibility? Is that because we need to allow overwriting it and someone else may implement the Checkbox differently?